### PR TITLE
Fix table display with long strings

### PR DESCRIPTION
**Pull request by artyom-savchenko**
Signed-off-by: Artyom Savchenko <artem.savchenko@xored.com>

Columns in table were stretched indefinitely when entering long strings:
Screen Shot 2021-02-28 at 16 08 52
I added support for max size and word wrap  in the table:
Screen Shot 2021-02-28 at 17 11 29

### CODE REVIEW

File: plugins/presentation/src/components/internal/presenters/value/StringEditor.svelte [hunk 1]
```diff
@@ -21,14 +21,24 @@
   export let attribute: AttrModel
   export let maxWidth: number = 300
   export let editable: boolean
+  export let textWrap: boolean
```
Comment (haiodo) [inline plugins/presentation/src/components/internal/presenters/value/StringEditor.svelte:24]: I do not think we should pass textWrap here, presenter should decide how to show data, it should not be controlled from outside, maxWidth is enough to decide I think.

**Comment by z2sx** [comment]
We want to have well-defined set of Presenter properties (which should make sense for every Presenter implementation). Looks like `max-width` should be handled by every presenter, so can we introduce `max-width` presenter property instead of passing it via `props`?

**Comment by artyom-savchenko** [comment]
I updated the MR, now the fix is not to ignore the max width in read only mode

**Comment by artyom-savchenko** [comment]
> We want to have well-defined set of Presenter properties (which should make sense for every Presenter implementation). Looks like `max-width` should be handled by every presenter, so can we introduce `max-width` presenter property instead of passing it via `props`?

Do you propose to add `max-width` property to `Presenter.svelte` add remove it from  presenters defined from classes?
At the moment 7 presenter has `max-width` property and its own implementation for it. Looks like it can be replaced by supporting `max-width` in `Presenter.svelte`.

**Comment by z2sx** [comment]
> Do you propose to add `max-width` property to `Presenter.svelte`

Yes, that's what I meant :)

**Comment by artyom-savchenko** [comment]
@z2sx @haiodo I updated the PR, please take a look.

**Review by haiodo — APPROVED** [review]

**Pull request merged**